src/elements/hx-accordion-panel/_shadow.scss

```diff
@@ -41,3 +41,8 @@
     display: block;
   }
 }
+
+button:focus {
+  outline: none;
+  box-shadow: 0 0 4px 0 #0e94a6;
+}
```

Comment:
Please move this into the Light DOM styles:

https://github.com/HelixDesignSystem/helix-ui/pull/675/files#diff-dc0ca2e5fac9b8928afe09ccdd3b84c0

This provides more flexibility for styling component states.